assets/src/blocks/Splittwocolumns/SplittwocolumnsFrontend.js

```diff
@@ -0,0 +1,104 @@
+import { Component, Fragment } from '@wordpress/element';
+
+const { apiFetch } = wp.apiFetch;
+
+export const SplittwocolumnsFrontend = (props) => {
+  const {
```

Comment:
You could define these directly instead of `props` in the previous line, makes it a bit cleaner 🙂